examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -84,12 +84,17 @@ void ModelCommand::CheckPeerICDType()
         return;
```

Comment:
This is intentional, since here we only care about the LIT vs. SIT & non-ICD.

Maybe what we need to change is the name of this function (how about `CheckPeerType`) instead of this log.